readme.md

```diff
@@ -170,6 +170,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [Perl 6](https://github.com/sfischer13/awesome-perl6) - Newest member of the Perl family.
```

Comment:
I don't feel this should assume knowledge of what Perl is. Perhaps something like "A new standard for Perl, a high-level interpreted scripting language" could be better.

Reply:
Thanks for the feedback and sorry for the delay. I adjusted the description.